docs/content/stable/yugabyte-platform/anywhere-automation/yb-kubernetes-operator.md

```diff
@@ -283,8 +283,6 @@ The operator pod (with the YugabyteDB Anywhere instance) should have the IAM rol
 
 The IAM role used should be sufficient to access storage in S3.
 
-To enable IAM roles to access storage in S3, set the **Use S3 IAM roles attached to DB node for Backup/Restore** Universe Configuration option (config key `yb.backup.s3.use_db_nodes_iam_role_for_backup`) to true. Refer to [Manage runtime configuration settings](../../administer-yugabyte-platform/manage-runtime-config/).
```

Comment:
This is false by default. Why are we removing this ? There can be client using YBA's IAM role. Do we need to set this to true by default ? Hi @agunta-yb please check this.